vmngclient/api/templates/__init__.py

```diff
@@ -28,6 +28,9 @@
 # CEdge Templates
 from vmngclient.api.templates.payloads.tenant.tenant_model import TenantModel
 
+# Cisco VPN Interface Ethernet Templates
+
```

Comment:
Are we missing import statement here?
`from vmngclient.api.templates.payloads.cisco_vmp_interface...`